bin/phpcs-changed.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# This is meant to run only if PHP files have been staged. It'll fatal as of now if phpcs-changed can't find php files in the diff.
+function phpcschanged {
+  # We need three files for phpcs-changed.
+  # 1. A file of the combined diff.
+  # 2. A file of the phpcs results for the original files.
+  # 3. A file of the phpcs results for the revised files.
+
+  # Make a folder for the files.
+  mkdir tmp-phpcschanged
```

Comment:
Do you think we could move this to a real `/tmp/` subdirectory, to make sure if anything happens, no stray files are left behind?

Reply:
I suppose we could. Are there any cases when someone is working on a machine without access to a `/tmp/` folder? Not sure about, say, a Chromebook.

Happy to make the change and face that if it comes up.

Reply:
Oh, that's a good question. After a bit of googling, it seems that the Chromebook also has that directory.

Reply:
53097bd60bf4d492df3b937ac1dffc4dc0df53d9